src/DIRAC/Core/Security/Locations.py

```diff
@@ -53,22 +53,38 @@ def getCAsLocation():
     casPath = "/etc/grid-security/certificates"
     if os.path.isdir(casPath):
         return casPath
-    # No CA's location found
-    return False
-
-
-# Retrieve CA's location
-
-
-def getCAsDefaultLocation():
-    """Retrievethe CAs Location inside DIRAC etc directory"""
     # rootPath./etc/grid-security/certificates
     casPath = f"{DIRAC.rootPath}/etc/grid-security/certificates"
-    return casPath
+    if os.path.isdir(casPath):
+        return casPath
+    return False
```

Comment:
It seems more natural to return `None` instead of `False` here, but I guess you want to avoid breaking the current  logic, don't you?

Reply:
It is a good suggestion, I applied it.